<launch>
    <node name="gripper_node" pkg="gripper" type="gripper_node.py" respawn="true" output="screen"/>
</launch>
